class.jetpack-sync.php

```diff
@@ -66,6 +66,45 @@ static function sync_options( $file, $option /*, $option, ... */ ) {
 		return call_user_func_array( array( $jetpack->sync, 'options' ), $args );
 	}
 
+	/**
+	 * @param string $option, Option name to sync
+	 * @param string $option ...
+	 */
+	static function sync_mock_option( $option, $callback ) {
+		if( is_network_admin() ) return;
+		$jetpack = Jetpack::init();
+		$args = func_get_args();
+		return call_user_func_array( array( $jetpack->sync, 'mock_option' ), $args );
+	}
+
+	/**
+	 * A static method to access the private
+	 *
+	 * @param  string $constant Constant that you would like to sync.
+	 *                The constant then becomes jetapack_CONSTANT_NAME. ( mock option )
+	 */
+	static function sync_constant( $constant ) {
+		if( is_network_admin() ) return;
+		$jetpack = Jetpack::init();
+		$args = array( $constant, array( 'Jetpack_Sync', 'get_sync_constant' ) );
+		return call_user_func_array( array( $jetpack->sync, 'mock_option' ), $args );
+	}
+
+	/**
+	 * Helper function to return the constants value.
+	 *
+	 * @param  string $constant [description]
+	 * @return value of the constant or '' if the constant is set to false or doesn't exits.
+	 */
+	static function get_sync_constant( $constant ) {
+
+		if ( defined( $constant ) && constant( $constant ) !== false ) {
+			return constant( $constant );
+		}
+
+		return ''; // we can't return false.
```

Comment:
I didn't want someone to be able to return a unpredictable value. - In the sense that if the value existed in the database it could return 'true'. But it is not likely to exist in the database and if someone really wanted to mess with it they could just over write the filter by making it run later.